integration_tests/e2e/activities/allocateToActivity.cy.ts

```diff
@@ -52,7 +58,7 @@ context('Allocate to activity', () => {
     cy.signIn()
   })
 
-  it('should click through allocate to activity journey', () => {
+  xit('should be able to allocate when selecting a specific start date', () => {
```

Comment:
what does 'xit' do? or is it a typo?

Reply:
My way of ignoring a test to speed up. My bad.

Reply:
Sorted @natclamp-moj